21/alpine/Dockerfile

```diff
@@ -5,6 +5,8 @@ ENV OTP_VERSION="21.1.1"
 RUN set -xe \
 	&& OTP_DOWNLOAD_URL="https://github.com/erlang/otp/archive/OTP-${OTP_VERSION}.tar.gz" \
 	&& OTP_DOWNLOAD_SHA256="a6588243e9659960e184083f49b15ea37890e4cd9bb8aef84ba4180db68befc3" \
+	&& REBAR3_VERSION="3.6.1" \
```

Comment:
I meant the `REBAR3_VERSION` can be in the `ENV` section, to be exposed to fianl user's environement

think about what can be useful to the final users, then expose, but keep the exposed ones as minimal

Reply:
something like this?

```dockerfile
ENV OTP_VERSION="..." \
	REBAR3_VERSION="3.6.1"
```

Reply:
I don't think the comments were resolved, I meant there are two kinds of variables:

1. shell variables, not keeping to end user environment
2. ENV variables, will be kept to end user visible environment

I feel only good to keep minimum meaningful variables to user visible environment,
and I feel good reasons to keep REBAR3_VERSION to be user visible

```dockerfile
ENV OTP_VERSION="..." \
	REBAR3_VERSION="3.6.1"
```

Similar to this line
https://github.com/erlang/docker-erlang-otp/blob/master/21/Dockerfile#L54